### ISSUE TITLE
Ping not supported anymore 

### ISSUE THREAD

**Issue by madmaxpt** [bwaldvogel/mongo-java-server#211]
Since the release 1.42.0, the following code 
```java
public static void main(String[] args)
	{
		var server = new MongoServer(new MemoryBackend());
		var serverAddress = server.bind();
		try(var client = createClient(serverAddress))
		{
			System.out.println(client.getDatabase("TEST").runCommand(new BsonDocument("ping", new BsonBoolean(true))));
		}
	}

	private static MongoClient createClient(InetSocketAddress serverAddress)
	{
		return MongoClients.create(
				MongoClientSettings.builder()
						.applyToClusterSettings(builder ->
								builder.hosts(Collections.singletonList(new ServerAddress(serverAddress))))
						.build());
	}
```
Provokes the following error

> Exception in thread "main" com.mongodb.MongoCommandException: Command failed with error 59 (CommandNotFound): 'no such command: 'ping'' on server view-localhost:55640. The full response is {"$err": "no such command: 'ping'", "errmsg": "no such command: 'ping'", "code": 59, "codeName": "CommandNotFound", "ok": 0}

Whereas it would previously work correctly and produce "Document{{ok=1.0}}"

It only works if the database name is "admin". However on a MongoDB server, the ping query described [here](https://www.mongodb.com/docs/manual/reference/command/ping/) produces an output, whatever the database name.

The Java MongoDB driver I used for this example is the following :

```xml
        <dependency>
            <groupId>org.mongodb</groupId>
            <artifactId>mongodb-driver-sync</artifactId>
            <version>4.7.0</version>
        </dependency>
```

**Comment by bwaldvogel** [comment]
Thanks for your bug report. It should be fixed via e9394f7.

**Comment by bwaldvogel** [comment]
The fix is available via `v1.44.0`
